app/database/item_store_integration_test.go

```diff
@@ -1074,18 +1074,18 @@ func TestItemStore_TriggerBeforeUpdate_SetsPlatformID(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			db := testhelpers.SetupDBWithFixtureString(`
 				platforms:
-					- {id: 3, regexp: "^1.*", priority: 1}
-					- {id: 4, regexp: "^2.*", priority: 2}
-					- {id: 2, regexp: "^1.*", priority: 3}
 					- {id: 1, regexp: "^4.*", priority: 4}
+					- {id: 2, regexp: "^1.*", priority: 3}
+					- {id: 4, regexp: "^2.*", priority: 2}
+					- {id: 3, regexp: "^1.*", priority: 1}
 				languages: [{tag: fr}]
 				items:
-					- {id: 1, platform_id: 4, url: 1234, default_language_tag: fr}`)
+					- {id: 1, platform_id: 1, url: 444, default_language_tag: fr}`)
 			defer func() { _ = db.Close() }()
 
 			itemStore := database.NewDataStore(db).Items()
-			assert.NoError(t, itemStore.ByID(1).UpdateColumn("platform_id", 4).Error())
```

Comment:
The reason was probably that because the `url` was changed during `SetupDBWithFixtureString`, it was probably resetted to its correct value "2".